https/kingpinflag/flag.go

```diff
@@ -20,7 +20,16 @@ import (
 // To use the default Kingpin application, call AddFlags(kingpin.CommandLine)
 func AddFlags(a *kingpin.Application) *string {
 	return a.Flag(
-		"web.config",
-		"Path to config yaml file that can enable TLS or authentication.",
+		"web.config.file",
```

Comment:
Do we want to change this? I don't want to break existing node_exporter users by changing the flag name.

Reply:
#21 will break node_exporter users :(

Note exporter can keep its current flag as hidden.

Reply:
Yea, good point, we can keep a compatibility flag alias.